src/content/docs/mobile-monitoring/new-relic-mobile-ios/watchOS/watchOS-agent-setup.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Starting the New Relic agent to your watchOS project
+tags:
+  - Mobile monitoring
+  - New Relic Mobile iOS
+  - Installation
+  - SwiftUI
+  - WatchOS
+metaDescription: How to properly setup your watchOS project with the New Relic agent.
+freshnessValidatedDate: never
+---
+
+WatchOS projects do not normally contain an ApplicationDelegate or ExtensionDelegate class. These delegates are necessary to configure and start the New Relic agent in a watchOS project. To add an AppDelegate to your watchOS application, follow the steps detailed here. Before adding the AppDelegate, follow the [outlined procedure](/docs/mobile-monitoring/new-relic-mobile-ios/installation/spm-installation) to correctly install the New Relic agent. 
```

Comment:
What do these classes do? Why are they a dependency? @mbruin-NR

Reply:
They are classes defined by protocols to be used with Apple APIs.
https://developer.apple.com/documentation/watchkit/wkapplicationdelegate
https://developer.apple.com/documentation/watchkit/wkextensiondelegate

Reply:
So is the purpose of this doc to show customers how to install an agent to a watchOS application, but to be able to do that they need to first add Apple APIs to their apps so that it's compatible with NR?

Reply:
I'm a little confused by the intro. It seems like they need to first install the agent, then add these classes... but the installation doesn't work _unless_ the APIs are added to their app?